erpnext/patches/v14_0/discount_accounting_separation.py

```diff
@@ -0,0 +1,8 @@
+import frappe
+
+def execute():
+    doc = frappe.get_doc("Accounts Settings")
+    discount_account = doc.enable_discount_accounting
```

Comment:
for patches this is fine but avoid getting whole doc just for single value. `get_single_value` or `get_value` should be used. 

reason `get_doc` is bad for performance.